extensions/stackdriver/metric/record.cc

```diff
@@ -61,6 +76,17 @@ void record(bool is_outbound, const ::wasm::common::NodeInfo &local_node_info,
       peer_rev_iter == peer_labels.end() ? kLatest : peer_rev_iter->second;
 
   if (is_outbound) {
+    std::string subset;
+    google::protobuf::Struct istio;
+    getMessageValue({"cluster_metadata", "filter_metadata", "istio"}, &istio);
```

Comment:
Can you pass “subset” in the initializer list ?
@kyessenov is that supported ?

Reply:
@gargnupur what happens if the destination cluster is empty ?
Will this call still get it or will it hit the optimization where cluster is not selected if it is empty.

Reply:
If destination cluster is empty then struct  will be empty...
If we check for return value here, it. should return false...